src/components/content/heroes/aurora/weather-easter-egg/weather-easter-egg.tsx

```diff
@@ -0,0 +1,7 @@
+import { flaresByWeather } from '../aurora-components';
+import { WeatherType } from '../aurora-types';
+import React from 'react';
+
+export const WeatherEasterEgg = ({ weather }) => {
```

Comment:
I am not completely persuaded by this, since the component doesnt have logic, but lets keep it like this for now